database-connector/src/main/java/life/qbic/projectmanagement/persistence/ExperimentalDesignRepository.java

```diff
@@ -13,9 +13,9 @@
 public class ExperimentalDesignRepository implements ExperimentalDesignLookup {
 
   @Override
-  public List<String> retrieveOrganisms() {
-    return Arrays.asList("Organism1", "Organism2", "Organism3",
-        "Organism4", "Organism5");
+  public List<String> retrieveSpecies() {
+    return Arrays.asList("Species1", "Species2", "Species3",
+        "Species4", "Species5");
   }
 
   @Override
```

Comment:
This was a test class from @Steffengreiner  right? This can be removed?

Reply:
has been removed with merge from master->dev->this PR